flutter/lib/src/feedback/sentry_feedback_widget.dart

```diff
@@ -360,8 +382,36 @@ class _SentryFeedbackWidgetState extends State<SentryFeedbackWidget> {
   void _dismiss({required bool pendingAssociatedEventId}) {
     SentryFeedbackWidget.pendingAssociatedEventId =
         pendingAssociatedEventId ? widget.associatedEventId : null;
+
+    _writePreservedData();
+
     if (mounted) {
       Navigator.maybePop(context);
     }
   }
+
+  void _restorePreservedData() {
+    final preservedName = SentryFeedbackWidget.preservedName;
+    if (preservedName != null) {
+      _nameController.text = preservedName;
+    }
+    final preservedEmail = SentryFeedbackWidget.preservedEmail;
+    if (preservedEmail != null) {
+      _emailController.text = preservedEmail;
+    }
+    final preservedMessage = SentryFeedbackWidget.preservedMessage;
+    if (preservedMessage != null) {
+      _messageController.text = preservedMessage;
+    }
+  }
+
+  void _writePreservedData() {
+    if (SentryFeedbackWidget.pendingAssociatedEventId != null) {
+      SentryFeedbackWidget.preservedName = _nameController.text;
+      SentryFeedbackWidget.preservedEmail = _emailController.text;
+      SentryFeedbackWidget.preservedMessage = _messageController.text;
+    } else {
+      SentryFeedbackWidget.clearPreservedData();
+    }
+  }
```

Comment:
we also have to clear it when `dispose()` is called right?

Reply:
So this would be in the case that we get killed by the system due to memory pressure or any other reason, right? Think it makes sense to reset in this case.